examples/dudect.rs

```diff
@@ -0,0 +1,18 @@
+//! This file collects Constant Time "benchmarks" using the `dudect-bencher` crate.
```

Comment:
So what's the reason this cannot live in `benches`?

Reply:
Short answer is "because it doesn't work!".

It can be an example or a binary, but when set up as a benchmark and ran with `cargo bench --bench dudectbench --features … …` it fails with an `error: Found argument '--bench' which wasn't expected, or isn't valid in this context`

I think the dudect-bencher crate is a bit old and un-loved and that if we take this on, we might have to do some work to fix a few quirks here and there.

I don't have a strong opinion between example or binary, either kind of suck.